modules/acquia_cms_common/config/optional/filter.format.filtered_html.yml

```diff
@@ -16,30 +16,29 @@ filters:
+      allowed_html: '<em> <strong> <cite> <blockquote cite> <code> <li> <dt> <dd> <img src alt data-entity-type data-entity-uuid data-align data-caption> <sup> <sub> <s> <u> <drupal-media data-entity-type data-entity-uuid data-view-mode data-align data-caption alt title> <a href hreflang class="coh-style-link-button-light coh-style-link-with-icon coh-style-link-button-dark coh-style-link-button-color"> <ul type class="coh-style-tick-list"> <ol start type class="coh-style-tick-list"> <p class="text-align-left text-align-center text-align-right text-align-justify coh-style-paragraph-extra-large coh-style-paragraph-small coh-style-paragraph-large"> <h2 id class="coh-style-heading-3-size coh-style-heading-2-size coh-style-pre-heading coh-style-pre-heading-dark coh-style-heading-5-size coh-style-pre-heading-blue coh-style-heading-1-size coh-style-heading-xl coh-style-heading-4-size coh-style-pre-heading-light"> <h3 id class="coh-style-heading-3-size coh-style-heading-2-size coh-style-pre-heading coh-style-pre-heading-dark coh-style-heading-5-size coh-style-pre-heading-blue coh-style-heading-1-size coh-style-heading-xl coh-style-heading-4-size coh-style-pre-heading-light"> <h4 id class="coh-style-heading-3-size coh-style-heading-2-size coh-style-pre-heading coh-style-pre-heading-dark coh-style-heading-5-size coh-style-pre-heading-blue coh-style-heading-1-size coh-style-heading-xl coh-style-heading-4-size coh-style-pre-heading-light"> <h5 id class="coh-style-heading-3-size coh-style-heading-2-size coh-style-pre-heading coh-style-pre-heading-dark coh-style-heading-5-size coh-style-pre-heading-blue coh-style-heading-1-size coh-style-heading-xl coh-style-heading-4-size coh-style-pre-heading-light"> <h6 id class="coh-style-heading-3-size coh-style-heading-2-size coh-style-pre-heading coh-style-pre-heading-dark coh-style-heading-5-size coh-style-pre-heading-blue coh-style-heading-1-size coh-style-heading-xl coh-style-heading-4-size coh-style-pre-heading-light"> <h1 class="coh-style-heading-3-size coh-style-heading-2-size coh-style-pre-heading coh-style-pre-heading-dark coh-style-heading-5-size coh-style-pre-heading-blue coh-style-heading-1-size coh-style-heading-xl coh-style-heading-4-size coh-style-pre-heading-light"> <dl class="coh-style-tick-list"> <span class="coh-style-paragraph-extra-large coh-style-link-button-light coh-style-heading-3-size coh-style-link-with-icon coh-style-heading-2-size coh-style-tick-list coh-style-pre-heading coh-style-pre-heading-dark coh-style-heading-5-size coh-style-link-button-dark coh-style-paragraph-small coh-style-pre-heading-blue coh-style-heading-1-size coh-style-heading-xl coh-style-heading-4-size coh-style-link-button-color coh-style-pre-heading-light coh-style-paragraph-large coh-color-brand-color-light coh-color-brand-color coh-color-brand-color-dark coh-color-gray coh-color-gray-10 coh-color-gray-20 coh-color-gray-30 coh-color-gray-70 coh-color-gray-80 coh-color-black coh-color-gray-90 coh-color-white">'
```

Comment:
@chandan-singh7929 Allowed html should contain html tags & html attributes. Any reason we are giving list of long list of different allowed classes ?
cc: @panshulK

Reply:
Hey @vishalkhode1 
The classes are getting added because of the change in the ckeditor text format toolbar - (We have now added Styles, Internal Styling, Element Styling) options in the ckeditor toolbar for Filtered Text format.
The classes are getting auto added once this is done.

Reply:
Ideally, adding `<drupal-media data-entity-type data-entity-uuid data-view-mode data-align data-caption alt title>` should do the work, this is the element get added when we embed entity.

Reply:
@vishalkhode1 , @chandan-singh7929 
Updated PR with latest changes. The extra classes added in the "Allowed html" section are now removed.